bad-aqua-config.ini

```diff
@@ -0,0 +1,14 @@
+[eth-network]
```

Comment:
please move this somewhere under tests or a samples folder. This file does not belong in the main codebase.